spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -372,7 +371,7 @@ def main(argv: Sequence[str]):
 
     file_out = os.path.abspath(arguments.o)
     append = bool(arguments.append)
-    if arguments.vert is not None:
+    if arguments.vertfile is not None:
```

Comment:
The case we want to adress here is if we provide vertebral labels (`vertfile`) without having to specify `vert` which corresponds to the levels we want to include. Before, since the condition was on `arguments.vert`, it puts directly the vertfile to None, even if it was specified by the user.

Reply:
Ahhh, I see. That makes sense.

Question: Do we need to have a conditional at all? If we just wrote:

```python
levels = arguments.vert
fname_vert_level = arguments.vertfile
```

And we made it  so that `arguments.vert` is `[]` by default (we would just have to add `default=''` to `arguments.vert`)...

Then the behavior would be identical, wouldn't it?